### PR TITLE
Services page

### PR DESCRIPTION
**Pull request by vpsx**
First pass over Treatments and Services content:
- "Determinants/Processes/Evaluation" columns diagram newly designed and SVG-ified
- Interactive mouse-over tabs added to same
- Determinants venn diagram made interactive with mouse-over info panel
- "Core Facilitation" diagram and "Taxonomy of Outcomes" diagrams reworked into plain HTML/React components
- Buttons to track reader to different headings
- Some layout adjustments

### CODE REVIEW

File: packages/client/src/assets/treatments_and_services/DPE_diagram.svg [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <text x="675" y="180">implementation</text>
+    <text x="675" y="205">success</text>
+    <text x="675" y="245">Assessment of</text>
+    <text x="675" y="270">health equity (e.g.,</text>
+    <text x="675" y="295">reach, retention)</text>
+
+    <!-- Flag text -->
+    <text x=" 55" y="385">Health Equity</text>
+    <text x=" 55" y="410">Implementation</text>
+    <text x=" 55" y="435">Framework</text>
```
Comment (gregfrasco) [inline packages/client/src/assets/treatments_and_services/DPE_diagram.svg]: There spaces in the x, should fix the positioning of the text
```suggestion
    <text x="55" y="385">Health Equity</text>
    <text x="55" y="410">Implementation</text>
    <text x="55" y="435">Framework</text>
```

File: packages/client/src/assets/treatments_and_services/DPE_diagram.svg [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    <!-- Title text -->
+    <g font-size="26" font-weight="bold">
+      <text x=" 30" y="50">Determinants</text>
+      <text x="330" y="50">Processes</text>
+      <text x="630" y="50">Evaluation</text>
+    </g>
+
+    <!-- Bullet text -->
+    <text x=" 75" y="168">Barriers</text>
+    <text x=" 75" y="258">Facilitators</text>
```
Comment (gregfrasco) [inline packages/client/src/assets/treatments_and_services/DPE_diagram.svg]: Same here
```suggestion
    <text x="75" y="168">Barriers</text>
    <text x="75" y="258">Facilitators</text>
```

File: packages/client/src/assets/treatments_and_services/DPE_diagram.svg [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    <!-- Flags -->
+    <path d="M   1 357 h 258 l 40 32 v 28 l -40 32 h -258 v -8 l 30 -24 v -28 l -30 -24 v -8 z" fill="#A888C7" />
+    <path d="M 301 357 h 258 l 40 32 v 28 l -40 32 h -258 v -8 l 30 -24 v -28 l -30 -24 v -8 z" fill="#FFD884" />
+    <path d="M 601 357 h 258 l 40 32 v 28 l -40 32 h -258 v -8 l 30 -24 v -28 l -30 -24 v -8 z" fill="#78CEE9" />
+  </g>
+
+  <g text-anchor="start" font-family="sans-serif" font-size="20">
+    <!-- Title text -->
+    <g font-size="26" font-weight="bold">
+      <text x=" 30" y="50">Determinants</text>
```
Comment (gregfrasco) [inline packages/client/src/assets/treatments_and_services/DPE_diagram.svg]: same here
```suggestion
      <text x="30" y="50">Determinants</text>
```

File: packages/client/src/assets/treatments_and_services/boards.gif
Comment (gregfrasco) [inline packages/client/src/assets/treatments_and_services/boards.gif:1]: This maybe able to be translated into react code better with Grids can Cards.

File: packages/client/src/assets/treatments_and_services/recovery_curve_graph.png
Comment (gregfrasco) [inline packages/client/src/assets/treatments_and_services/recovery_curve_graph.png:1]: This is a little blurry, maybe that something we can fix next time, or if they have a different image

**Review by gregfrasco — CHANGES_REQUESTED** [review]
Going to pull down and look at the pages, but here are some comments for now

**Comment by gregfrasco** [comment]
The page itself looks good.

File: packages/client/src/assets/treatments_and_services/DPE_diagram.svg (hunk 1 above)
Comment (vpsx) [reply packages/client/src/assets/treatments_and_services/DPE_diagram.svg]: Wooowww sorry thank you :joy: :facepalm: :facepalm: I fixed it in the code/components version but of course forgot to do it here...

File: packages/client/src/assets/treatments_and_services/boards.gif
Comment (vpsx) [reply packages/client/src/assets/treatments_and_services/boards.gif:1]: Ah in the last meeting they said they were going to throw out this image/thing and find a 'more interesting' image so I told them I would hold off on this one!

File: packages/client/src/assets/treatments_and_services/recovery_curve_graph.png
Comment (vpsx) [reply packages/client/src/assets/treatments_and_services/recovery_curve_graph.png:1]: SVG'd it!

**Review by gregfrasco — APPROVED** [review]

**Pull request merged**